### DISCUSSION TITLE
Open kitty here with Dolphin services

### DISCUSSION THREAD

**Discussion by ghost** [kovidgoyal/kitty#4233] [Q&A] [answered]
Hello everyone, can anyone help me ?

I'm trying to add "Open Kitty here " as Dolphin services in Kubuntu 20.04.3 LTS

The `kittyhere.desktop`i have here. 
```
[Desktop Entry]
Type=Service
X-KDE-ServiceTypes=KonqPopupMenu/Plugin
MimeType=inode/directory;
Actions=openKittyHere;
X-KDE-AuthorizeAction=shell_access

[Desktop Action openKittyHere]
TryExec=konsole
Icon=utilities-terminal
Exec= ~./local/kitty.app/bin/kitty --directory %f
Name=Open Kitty Here

```
I'm not using `sudo apt-get install kitty`, i'm just using `curl` method to get latest version. But i can't make it work with.
Anyone have resolve way?

**Comment by page-down** [comment, marked as answer]
You should change TryExec to the path of kitty. Then make sure the path in Exec is correct.

It will be easier to execute if you create kitty symbolic link in $PATH.

If you want to use the kitty icon, you can put the icon file in ~/.local/share/icons/... , or specify the absolute path.

```text
[Desktop Entry]
Type=Service
Icon=kitty
X-KDE-ServiceTypes=KonqPopupMenu/Plugin
MimeType=inode/directory;
Actions=openKittyHere
X-KDE-AuthorizeAction=shell_access

[Desktop Action openKittyHere]
Name=Open kitty Here
Icon=kitty
TryExec=kitty
Exec=kitty --single-instance --directory %f
```

**Reply by ghost** [reply]
@page-down Thanks for recommend, i'm following every step in this [tutorial](https://sw.kovidgoyal.net/kitty/binary/). But look like i'm making a mess atm now
Here my `echo $PATH`
```
/home/hoangtho/.local/kitty.app/bin /usr/local/sbin /usr/local/bin /usr/sbin /usr/bin /sbin /bin /usr/games /usr/local/games /snap/bin
```

**Reply by ghost** [reply]
@page-down Hey . Thanks for your help. My problem is solved. Look like im linking wrong folder ^^. Thanks for your help ^^. Problem is solved !

**Comment by Luflosi** [comment]
Would it make sense to make this into a PR so other people can benefit from this as well?

**Reply by page-down** [reply]
I'm not sure, because this is something that is more of a personal desktop environment customization. There are just so many ways to integrate with other software or DE, and having only one menu integration instruction for KDE is clearly not enough and seems odd. I think GitHub Discussions would be a good fit for now. :)

In KDE, you can set kitty as the default terminal right in the GUI settings.
